sdk/RELEASE.md

```diff
@@ -9,6 +9,20 @@
 
 ## Deprecations
 
+## Bug Fixes and Other Changes
+
+## Documentation Updates
+# 1.8.17
+## Major Features and Improvements
+
+## Breaking Changes
+
+### For Pipeline Authors
+
+### For Component Authors
+
+## Deprecations
+
 ## Bug Fixes and Other Changes
 * Upgrade kubernetes library to v19 to get access to EphemeralVolume [#8526](https://github.com/kubeflow/pipelines/pull/8526) 
```

Comment:
Change log for https://github.com/kubeflow/pipelines/pull/8543?

Reply:
and https://github.com/kubeflow/pipelines/pull/8428

Reply:
Added, thank you

Reply:
Still missing changelog for #8428 ?

Reply:
I added above in "for pipeline authors". Do you think this should be in "other changes" instead?

```markdown
### For Pipeline Authors
* Allow ephemeral storage to be set via component output parameter [#8428](https://github.com/kubeflow/pipelines/pull/8428)
```